internal/controllers/suite_test.go

```diff
@@ -65,29 +48,51 @@ func TestMain(m *testing.M) {
 	}
 
 	cfg, err := testEnv.Start()
-	if err != nil {
-		fmt.Println(err)
-		os.Exit(1)
+	utilruntime.Must(err)
+	if cfg == nil {
+		log.Panic("expected cfg to not be nil")
 	}
 
 	sch = runtime.NewScheme()
 	utilruntime.Must(operatorsv1alpha1.AddToScheme(sch))
 	utilruntime.Must(rukpakv1alpha1.AddToScheme(sch))
 
 	cl, err = client.New(cfg, client.Options{Scheme: sch})
-	if err != nil {
-		fmt.Println(err)
-		os.Exit(1)
+	utilruntime.Must(err)
+	if cl == nil {
+		log.Panic("expected cl (client.New) to not be nil")
 	}
 
 	code := m.Run()
 
-	// tearing down the test environment
-	err = testEnv.Stop()
-	if err != nil {
-		fmt.Println(err)
-		os.Exit(1)
+	// Test Cleanup
```

Comment:
echoing the comment from before - we're in an integration test environment that has a lifetime scoped to the lifetime of tests for this package - I don't know that there is sufficient motivation to doing "cleanup" in this situation